gallery/src/components/NxStatefulTabs/NxStatefulTabsSimpleExample.tsx

```diff
@@ -21,10 +21,10 @@ const NxStatefulTabsSimpleExample = () => {
   return (
     <NxStatefulTabs>
       <NxTabList>
-        <NxTab id="tab-1" onClick={clickHandler}>Tab 1</NxTab>
-        <NxTab id="tab-2" onClick={clickHandler}>Tab 2</NxTab>
-        <NxTab id="tab-3" onClick={clickHandler}>Tab 3</NxTab>
-        <NxTab id="tab-4" onClick={clickHandler}>Tab 4</NxTab>
+        <NxTab id="tab-1" tabIndex={1} onClick={clickHandler}>Tab 1</NxTab>
+        <NxTab id="tab-2" tabIndex={2} onClick={clickHandler}>Tab 2</NxTab>
+        <NxTab id="tab-3" tabIndex={3} onClick={clickHandler}>Tab 3</NxTab>
+        <NxTab id="tab-4" tabIndex={4} onClick={clickHandler}>Tab 4</NxTab>
```

Comment:
Using tabIndexes other than 0 is not recommended.  0 means just put them in normal tab order.
```suggestion
        <NxTab id="tab-1" tabIndex={0} onClick={clickHandler}>Tab 1</NxTab>
        <NxTab id="tab-2" tabIndex={0} onClick={clickHandler}>Tab 2</NxTab>
        <NxTab id="tab-3" tabIndex={0} onClick={clickHandler}>Tab 3</NxTab>
        <NxTab id="tab-4" tabIndex={0} onClick={clickHandler}>Tab 4</NxTab>
```

Reply:
ceabd4e